kivy/parser.py

```diff
@@ -150,5 +150,13 @@ def parse_float4(text):
     return value
 
 
-parse_int = int
-parse_float = float
+def parse_int(text):
```

Comment:
@akshayaurora Do you mean that wrapping `int()` and `float()` to functions? That's actually the point, because then It'll highlight them as the remaining functions. Otherwise it'll be like [this.](https://kivy.org/docs/api-kivy.parser.html#kivy.parser.parse_int)